pkg/resources/notification_integration.go

```diff
@@ -125,7 +125,8 @@ func NotificationIntegration() *schema.Resource {
 		Read:   ReadNotificationIntegration,
 		Update: UpdateNotificationIntegration,
 		Delete: DeleteNotificationIntegration,
-		Exists: NotificationIntegrationExists,
+		//lintignore:R003
+		Exists: CheckNotificationIntegration,
```

Comment:
**[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>R003: resource should not include Exists function